.github/workflows/deployDev.yml

```diff
@@ -53,11 +53,16 @@ jobs:
       - name: Install dependencies
         run: yarn install
       - name: Build deploy artifact
+        env: 
+          REACT_APP_BASE_URL: https://dev.simplereport.gov
+          REACT_APP_APPINSIGHTS_KEY: 29a75c50-8485-4d80-8e1b-41b1a091abb6
```

Comment:
We could make this a secret, not to have it actually be secret (a bit late) but to reduce the number of copies of it.

Reply:
I think this key might be unique per enviornment. I think the issue we had where local logs were in the prod instance was due to this key being the same